xbmc/filesystem/CurlFile.h

```diff
@@ -209,5 +211,6 @@ namespace XFILE
       MAPHTTPHEADERS m_requestheaders;
 
       long            m_httpresponse;
+      bool            m_isshoutcast;
```

Comment:
Yet the test for a Shoutcast, and the announcement to debug log is done in CCurFile. Why not also annouce this to other parts of the processing where it can be useful too?